onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -645,6 +647,15 @@ def get_args():
         help="Block size to quantize with. See https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py for details.",
     )
 
+    blockwise_group.add_argument(
+        "--int4_accuracy_level",
+        required=False,
```

Comment:
Can you add a default value for `int4_accuracy_level`?

Reply:
the default is unspecified, i.e., whatever MatMulNBits has as the accuracy_level default attribute value. I opted to not add a default here so we wouldn't need to keep them in sync.